src/metrics.cpp

```diff
@@ -302,8 +307,8 @@ int printStats(bool isScreen, bool mining)
             if (netheight <= 0)
                 netheight = 1;
             int downloadPercent = height * 100 / netheight;
-            std::cout << "     " << _("Downloading blocks") << " | " << height << " (" << nHeaders << " " << _("headers") << ") / ~" << netheight << " (" << downloadPercent << "%)" << std::endl;
 
+            std::cout << "     " << _("Downloading blocks") << " | " << height << " (" << nHeaders << " " << _("headers") << ") / ~" << netheight << " (" << downloadPercent << "%)" << std::endl;
```

Comment:
This diff is confusing; I think the blank line was moved up one line? Would be nice to undo this change.